docs/translations.rst

```diff
@@ -6,6 +6,38 @@ localization strings. For more comprehensive information, consult the
 `Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_
 or `this helpful blog post <http://blog.bessas.me/post/65775299341/using-gettext-in-django>`_.
 
+How translations work
+---------------------
+There are three major components to the translations system - identifying
+strings that need to be translated, having human translators write translations
+for those strings, and then actually inserting those translations when
+appropriate.
+
+1. ``./manage.py makemessages`` goes over the source code and pulls out strings
+   that have been tagged for translation. It does not actually execute code, so
+   you really can't do anything fancy in these strings.
+2. Transifex is a third party site we use for translating these strings. Human
+   translators look at the strings generated by makemessages and type up
+   appropriate translations. They have no context beyond what's in the string
+   itself. Be nice to them and give template variables names, so instead of "{}
+   version" they see "{build_date} version", which will be easier to understand.
```

Comment:
The first example (`instead of "{}...`) is obviously `str.format()` syntax, but the second (`see "{build_date}...`) could be either that or an f-string. If the difference matters, it would be nice to clarify that here.

Reply:
On review review, maybe you were implying `"{build_date} version"` is _not_ an f-string (because you didn't put on a leading `f`)?  I read this as the quotes referring to exactly what the translator sees as the "body of the thing to be translated".

Reply:
I meant for both to be `str.format`, but that's secondary to the point.  What I wanted to convey was that translators will _only_ see the string template, not the surrounding code or the page.  `"{x} version"` is similarly inscrutable.  Any suggestions for how to phrase that without getting too long-winded here?

Reply:
Hmm, I think if this doc outlines (somewhere earlier) that f-strings must _not_ be used ever (because they will be evaluated prior to translation lookup), then it's fine to leave this as implying `str.format()`.

Reply:
I updated that text, lemme know what you think